sdk.mjs

```diff
@@ -40,29 +39,14 @@ const TITLE_TO_SLUG_RANGE_REGEX = /[^a-z\d]/g;
  */
 export const SVG_PATH_REGEX = /^m[-mzlhvcsqtae\d,. ]+$/i;
 
-/**
- * Get the directory name where this file is located from `import.meta.url`,
- * equivalent to the `__dirname` global variable in CommonJS.
- * @param {string} importMetaUrl Relative `import.meta.url` value of the caller.
- * @returns {string} Directory name in which this file is located.
- */
-export const getDirnameFromImportMeta = (importMetaUrl) =>
-	path.dirname(fileURLToPath(importMetaUrl));
-
 /**
  * Build a regex to validate HTTPs URLs.
- * @param {string} [jsonschemaPath] Path to the *.jsonschema.json* file.
  * @returns {Promise<RegExp>} Regex to validate HTTPs URLs.
  */
-export const urlRegex = async (
-	jsonschemaPath = path.join(
-		getDirnameFromImportMeta(import.meta.url),
-		'.jsonschema.json',
-	),
-) =>
+export const urlRegex = async () =>
```

Comment:
I think that this function `urlRegex` can be moved to *scripts/add-icon-data.js*, which is the unique script that is using it. Could you do that on this PR and run again `node scripts/release/update-sdk-ts-defs.js`?

Reply:
Done!